src/detections/filters/accordion-actions/add-filter.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Button } from '@material-ui/core'
+import AddIcon from '@material-ui/icons/Add'
+
+import { useIsDisabled } from 'detections/filters/hooks/use-is-disabled'
+import { useAddFilterComponent } from 'detections/filters/hooks/use-add-filter-component'
+
+export const AddFilter = () => <Button fullWidth variant={'contained'} color={'primary'}
```

Comment:
Whenever we've got to pass so many props that we can't fit them on a single line, we should extract them into a separate `useProps` hook.

Reply:
done